external/leveldb/LevelDbTuning.ipynb

```diff
@@ -761,8 +337,10 @@
     "- <https://wiesen.github.io/post/leveldb-storage-memtable/>\n",
     "- <https://www.igvita.com/2012/02/06/sstable-and-log-structured-storage-leveldb/>"
    ],
-   "cell_type": "markdown",
-   "metadata": {}
+   "cell_type": "code",
+   "metadata": {},
+   "execution_count": null,
+   "outputs": []
   },
   {
    "cell_type": "code",
```

Comment:
Can you please remove these empty cells at the end?
I think I added one of them in my "cleanup" PR, but they're adding an empty black box at the bottom of the rendered notebook.